From cccd334562329858feac719ad94b75aa87968a99 Mon Sep 17 00:00:00 2001
From: Britt Treece <britt.treece@gmail.com>
Date: Mon, 16 Jul 2018 04:59:03 -0500
Subject: [PATCH] add file backend to quick start guide (#727)

* add file backend to quick start guide

* add formatting to file backend debug call

* fix my poorly exec'ed fork update
---
 backends/file/client.go | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/backends/file/client.go b/backends/file/client.go
index f7c4460..c37684a 100644
--- a/backends/file/client.go
+++ b/backends/file/client.go
@@ -112,7 +112,7 @@ func (c *Client) watchChanges(watcher *f
 		for {
 			select {
 			case event := <-watcher.Events:
-				log.Debug("event:", event)
+				log.Debug("event: %v", event)
 				if event.Op&fsnotify.Write == fsnotify.Write ||
 					event.Op&fsnotify.Remove == fsnotify.Remove ||
 					event.Op&fsnotify.Create == fsnotify.Create {
